packages/e2e/src/specs/messages/messages-search.e2e.ts

```diff
@@ -0,0 +1,368 @@
+    await waitForClickableAndClick(await navigationTabs.messagesTab)
+  })
+  it("Search messages by NOT existing contact name and check results", async () => {
+    await browser.pause(4000)
+    await messagesPage.insertTextToSearchInput(contacts[4].firstName)
+    const resultList = await messagesPage.searchResultsOverlayList
+    const emptyList = await messagesPage.emptySearchResultsOverlayList
+    await expect(resultList).toBeDisplayed()
+    await expect(emptyList).toBeDisplayed()
+  })
+
+  it("Search messages by NOT existing surname and check results", async () => {
+    await messagesPage.insertTextToSearchInput(contacts[5].lastName)
+    const resultList = await messagesPage.searchResultsOverlayList
+    const emptyList = await messagesPage.emptySearchResultsOverlayList
+    await expect(resultList).toBeDisplayed()
+    await expect(emptyList).toBeDisplayed()
+  })
+
+  it("Search messages by partial NOT existing contact name and check results", async () => {
+    await messagesPage.insertTextToSearchInput(
+      contacts[5].firstName.slice(0, 4)
+    )
+    const resultList = await messagesPage.searchResultsOverlayList
+    const emptyList = await messagesPage.emptySearchResultsOverlayList
+    await expect(resultList).toBeDisplayed()
+    await expect(emptyList).toBeDisplayed()
+  })
+
+  it("Search messages by contact number and check results", async () => {
+    await messagesPage.insertTextToSearchInput(contacts[5].primaryNumber)
+    const resultList = await messagesPage.searchResultsOverlayList
+    const emptyList = await messagesPage.emptySearchResultsOverlayList
+    await expect(resultList).toBeDisplayed()
+    await expect(emptyList).not.toBeDisplayed()
+  })
+
+  it("Search messages by partial contact number and check results", async () => {
+    await messagesPage.insertTextToSearchInput(
+      contacts[5].primaryNumber.slice(0, 3)
+    )
+    const resultList = await messagesPage.searchResultsOverlayList
+    const emptyList = await messagesPage.emptySearchResultsOverlayList
+    await expect(resultList).toBeDisplayed()
+    await expect(emptyList).not.toBeDisplayed()
+  })
+})
+
+describe("Messasges search scenarios - results overlay 'SEE ALL'", () => {
+  before(async () => {
+    await waitForClickableAndClick(await navigationTabs.contactsTab)
+    await waitForClickableAndClick(await navigationTabs.messagesTab)
+  })
+
+  it("Click SEE ALL on search results overlay", async () => {
+    await messagesPage.insertTextToSearchInput(commonword)
+    const resultList = await messagesPage.searchResultsOverlayList
+    const emptyList = await messagesPage.emptySearchResultsOverlayList
+
+    await expect(resultList).toBeDisplayed()
+    await expect(emptyList).not.toBeDisplayed()
+
+    await waitForClickableAndClick(await messagesPage.seeAllSearchResultsButton)
+
+    const searchResultText = await messagesPage.searchResultsForText
+    const expectedText = 'Search results for "' + commonword + '"'
```

Comment:
Please, use template string :)

Reply:
it was unnecessary line + next one console log, remains of the fight with quotes in the 'expect' method https://github.com/mudita/mudita-center/pull/1172/commits/4350a74f4d4aeffca636b038de09a742cba2fae9